docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3501,6 +3501,89 @@ Return low-level information on the node `id`
 -   **404** – no such node
 -   **500** – server error
 
+### Remove a node
+
+
+`DELETE /nodes/<id>`
+
+Remove a node [`id`] from the Swarm.
+
+**Example request**:
+
+    DELETE /nodes/24ifsmvkjbyhk HTTP/1.1
+
+**Example response**:
+
+    HTTP/1.1 200 OK
+    Content-Length: 0
+    Content-Type: text/plain; charset=utf-8
+
+**Query parameters**:
+
+-   **force** - 1/True/true or 0/False/false, Force remove an active node.
+        Default `false`.
+
+**Status codes**:
+
+-   **200** – no error
+-   **404** – no such node
+-   **500** – server error
+
+### Update a node
+
+
+`POST /nodes/<id>/update`
+
+Update the node `id`.
+
+The payload of the `POST` request is the new `NodeSpec` and will
+override the current `NodeSpec` for the specified node.
+
+If `Availability` or `Role` are omitted, this will return an
+error. Any other field omitted will reset the current value to either
+an empty value or the default cluster-wide value.
```

Comment:
Trying to prevent usage of future tense;

```Markdown
The payload of the `POST` request is the new `NodeSpec` and
overrides the current `NodeSpec` for the specified node.

If `Availability` or `Role` are omitted, this returns an
error. Any other field omitted resets the current value to either
an empty value or the default cluster-wide value.
```